schemas/audiences/segment.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/audiences/segment",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Assignment",
+  "type": "object",
+  "description":
+    "",
+  "definitions": {
+    "segment": {
+      "properties": {
```

Comment:
I think a `xdm:confidence` property is missing here to capture the confidence of the segment assignment, as outlined in #110